<def f='boost-1.85.0/libs/exception/include/boost/exception/detail/exception_ptr.hpp' l='507' ll='528' type='boost::exception_ptr boost::current_exception()'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='461' u='c' c='_ZN5boost6detail17shared_state_base23mark_exceptional_finishEv'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='1066' u='c' c='_ZN5boost6detail28future_deferred_shared_state7executeERNS_11unique_lockINS_5mutexEEE'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='1088' u='c' c='_ZN5boost6detail28future_deferred_shared_stateIRT_T0_E7executeERNS_11unique_lockINS_5mutexEEE'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='1115' u='c' c='_ZN5boost6detail28future_deferred_shared_stateIvT_E7executeERNS_11unique_lockINS_5mutexEEE'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='3085' u='c' c='_ZN5boost6detail17task_shared_state8do_applyEv'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='3165' u='c' c='_ZN5boost6detail17task_shared_stateIT_RT0_E8do_applyEv'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='3246' u='c' c='_ZN5boost6detail17task_shared_stateIPFT_vES2_E8do_applyEv'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='3324' u='c' c='_ZN5boost6detail17task_shared_stateIPFRT_vES3_E8do_applyEv'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='3399' u='c' c='_ZN5boost6detail17task_shared_stateIT_vE8do_applyEv'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='3474' u='c' c='_ZN5boost6detail17task_shared_stateIPFvvEvE8do_applyEv'/>
<use f='boost-1.85.0/libs/thread/include/boost/thread/future.hpp' l='4433' u='c' c='_ZN5boost23make_exceptional_futureEv'/>
